person.rb

```diff
@@ -0,0 +1,29 @@
+require_relative './nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    super()
+  end
```

Comment:
- [ ] A Person should have rentals. When the class is initialized there should be an empty rentals array `(@rentals = [])`. Kindly add it here.